zipline/pipeline/engine.py

```diff
@@ -330,6 +329,11 @@ def run_chunked_pipeline(self, pipeline, start_date, end_date, chunksize):
         )
         chunks = [self.run_pipeline(pipeline, s, e) for s, e in ranges]
 
+        if len(chunks) == 1:
```

Comment:
We'd care about this if we add a default (large-ish) chunksize on the Q research environment.